deploy/ansible/roles/certbot/tasks/main.yml

```diff
@@ -11,4 +11,5 @@
     name: Certbot automatic renewal.
     job: "/usr/bin/certbot renew --quiet --no-self-upgrade && service nginx restart"
     minute: 0
-    hour: 23
\ No newline at end of file
+    hour: 23
+    
```

Comment:
This file still lacks a trailing newline

Reply:
Gah, I could have sword I caught all those!  Thank you :)  Commit with fix: https://github.com/bocoup/people/pull/55/commits/288679cd883dd6b6666449b848416f83c00effcd